src/Accounts/AssemblyLoading/ConditionalAssemblyProvider.cs

```diff
@@ -42,11 +42,11 @@ public static void Initialize(string rootPath, IConditionalAssemblyContext conte
                 // todo: add a tool to update assembly versions after replacing the assemblies. (Can it support newly introduced assemblies?)
                 // todo: consider moving the list to a standalone config file
                 #region AssemblyList
-                CreateAssembly("netcoreapp2.1", "Azure.Core", "1.35.0.0").WithPowerShellCore(),
-                CreateAssembly("netcoreapp2.1", "Microsoft.Identity.Client", "4.49.1.0").WithPowerShellCore(),
-                CreateAssembly("netcoreapp3.1", "Microsoft.Identity.Client.Extensions.Msal", "2.23.0.0").WithPowerShellCore(),
+                CreateAssembly("netcoreapp2.1", "Azure.Core", "1.36.0.0").WithPowerShellCore(),
+                CreateAssembly("netcoreapp2.1", "Microsoft.Identity.Client", "4.56.0.0").WithPowerShellCore(),
+                CreateAssembly("netcoreapp3.1", "Microsoft.Identity.Client.Extensions.Msal", "4.56.0.0").WithPowerShellCore(),
```

Comment:
Are these version necessary?

Reply:
There are not necessary, reverting these. Thanks for catching this.